pages/common/wget.md

```diff
@@ -5,24 +5,28 @@
 
 - Download the contents of an URL to a file:
 
-`wget -O {{filename}} {{url}}`
+`wget {{https://example.com}}`
 
-- Limit download speed:
+- Download a single web page and all its associated resources:
 
-`wget --limit-rate={{200k}} {{url}}`
+`wget --page-requisites --convert-links {{https://example.com/somepage.html}}`
 
-- Continue an incomplete download:
+- Download a full website, with 3-second intervals between requests:
+
+`wget --mirror --page-requisites --convert-links --wait=3 {{https://example.com}}`
 
-`wget -c {{url}}`
+- Download the contents of an URL via authenticated FTP:
 
-- Download a full website:
+`wget --ftp-user={{username}} --ftp-password={{password}} {{ftp://example.com}}`
 
-`wget --mirror -p --convert-links -P {{target_folder}} {{url}}`
+- Limit download speed:
```

Comment:
Maybe better to mention the speed too ?
`Limit download speed to 200kBps`.

Because just 200k might give the impression of size, not speed.

Reply:
> The original example had a "target_folder" too. Are you sure we don't need it anymore ?

(replying in the correct location, for clarity)

We don't *strictly* need it, as by default the url is used as the folder name. Of course, if we decide that a custom name is an important part of this example, then the `-P` is needed in that sense. However, as I mentioned in the commit message, I deliberately chose to "remove -O and -P, since the target files/folders can be easily renamed using standard tools; this leaves space for more tool-specific options."

Reply:
Alright, so then lets mention that the site will be downloaded to a folder with the same name as url, for extra clarity.

Reply:
Wouldn't that make it too long, though? The description currently says:

> Download a full website, with 3-second intervals between requests:

Reply:
Right, I see that you have removed the `-O` option from the first example. How about we mention this info in the first example, and keep this as is ? So this does not increase in length and the user can deduce that the same file output rules apply.

Reply:
Good point. I'm not sure how to say it compactly, though, because the default filename will either be the last part of the URL (after the last slash), or default to `index.html` if pointing directly to the root domain of a website (that is, assuming I'm not missing any edge cases).

Reply:
We can add a url path to the root domain like this - 'http://example.com/foo' and say that the downloaded file name will be foo. The users can deduce that for the root domain it will be `index.html`.

Reply:
Ok 👍